identify/extensions.py

```diff
@@ -4,62 +4,138 @@
 
 
 EXTENSIONS = {
+    'apinotes': {'text', 'apinotes'},
+    'asar': {'binary', 'asar'},
+    'bash': {'text', 'shell', 'bash'},
     'bat': {'text', 'batch'},
     'bz2': {'binary', 'bzip2'},
     'c': {'text', 'c'},
     'cc': {'text', 'c++'},
+    'cfg': {'text', 'conf'},
```

Comment:
I was meaning towards omission as well (why I didn't fill out  . txt).  Maybe we just leave these as text and give higher priority to the named files?